spec/rubocop/cop/internal_affairs/redundant_message_argument_spec.rb

```diff
@@ -6,59 +6,72 @@
   context 'when `MSG` is passed' do
     it 'registers an offense' do
       expect_offense(<<-RUBY, 'example_cop.rb')
-      add_offense(node, :expression, MSG)
-                                     ^^^ Redundant message argument to `#add_offense`.
+      add_offense(node, message: MSG)
+                        ^^^^^^^^^^^^ Redundant message argument to `#add_offense`.
       RUBY
     end
 
     it 'auto-corrects' do
-      new_source = autocorrect_source('add_offense(node, :expression, MSG)')
+      new_source = autocorrect_source('add_offense(node, message: MSG)')
 
-      expect(new_source).to eq('add_offense(node, :expression)')
+      expect(new_source).to eq('add_offense(node)')
     end
   end
 
   it 'does not register an offense when formatted `MSG` is passed' do
     expect_no_offenses(<<-RUBY, 'example_cop.rb')
-      add_offense(node, :expression, MSG % foo)
+      add_offense(node, location: :expression, message: MSG % foo)
     RUBY
   end
 
   context 'when `#message` is passed' do
     it 'registers an offense' do
       expect_offense(<<-RUBY, 'example_cop.rb')
-      add_offense(node, :expression, message)
-                                     ^^^^^^^ Redundant message argument to `#add_offense`.
+      add_offense(node, location: :expression, message: message)
+                                               ^^^^^^^^^^^^^^^^ Redundant message argument to `#add_offense`.
       RUBY
     end
 
     it 'auto-corrects' do
-      new_source = autocorrect_source('add_offense(node, :expression, message)')
+      new_source = autocorrect_source(<<-RUBY)
+        add_offense(
+          node,
+          location: :expression,
+          message: message,
+          severity: :error
+        )
+      RUBY
 
-      expect(new_source).to eq('add_offense(node, :expression)')
+      expect(new_source).to eq(<<-RUBY)
+        add_offense(
+          node,
+          location: :expression,
+          severity: :error
+        )
+      RUBY
     end
   end
 
   context 'when `#message` with offending node is passed' do
     it 'registers an offense' do
       expect_offense(<<-RUBY, 'example_cop.rb')
-      add_offense(node, :expression, message(node))
-                                     ^^^^^^^^^^^^^ Redundant message argument to `#add_offense`.
+      add_offense(node, message: message(node))
+                        ^^^^^^^^^^^^^^^^^^^^^^ Redundant message argument to `#add_offense`.
       RUBY
     end
 
     it 'auto-corrects' do
       new_source =
-        autocorrect_source('add_offense(node, :expression, message(node))')
+        autocorrect_source('add_offense(node, message: message(node))')
```

Comment:
We might want to test the auto-correct when there are additional arguments, to ensure that there are no syntax errors.